datadog.go

```diff
@@ -125,9 +125,7 @@ func sanitizeMetricName(namespace string, v *view.View) string {
 func viewSignature(namespace string, v *view.View) string {
 	var buf strings.Builder
 	buf.WriteString(sanitizeMetricName(namespace, v))
-	for _, k := range v.TagKeys {
-		buf.WriteString("_" + k.Name())
-	}
+
 	return buf.String()
```

Comment:
If we do this (see https://github.com/DataDog/opencensus-go-exporter-datadog/pull/61/files#r390162410) we should just get rid of the buffer `buf` and simply:
```
return sanitizeMetricName(namespace, v)
```